### PR TITLE
docs: SEO improvements

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
This pull request is intended to optimize the SEO of the term _service mesh_. 

[SEO Plan Document](https://docs.google.com/document/d/1ukeEBZwNAFHMF_qK1wReIf22pVRSdhIRcwmXHmUaGHk/edit#heading=h.536ulhwwbpqa)

🔍  [Preview](https://consul-qhqakljz7-hashicorp.vercel.app/docs/intro/usecases/what-is-a-service-mesh)

### CODE REVIEW

File: website/content/docs/intro/usecases/what-is-a-service-mesh.mdx
Comment (blake) [inline website/content/docs/intro/usecases/what-is-a-service-mesh.mdx:15]:
```suggestion
A service mesh provides benefits for all organizations, ranging from security to improved application resiliency.
```
Is there a reason that service mesh is italicized throughout the entire document?

File: website/content/docs/intro/usecases/what-is-a-service-mesh.mdx
Comment (blake) [inline website/content/docs/intro/usecases/what-is-a-service-mesh.mdx:87]:
```suggestion
For Kubernetes, most service mesh can be installed by operators through a [Helm chart](https://helm.sh/). Additionally, the service mesh may offer a CLI tool that supports the installation and maintenance of the service mesh.
```
It might be better to link to the Helm website so that the reader can better understand what Helm is before exploring a chart registry like ArtifactHub.

File: website/content/docs/intro/usecases/what-is-a-service-mesh.mdx
Comment (blake) [inline website/content/docs/intro/usecases/what-is-a-service-mesh.mdx:24]:
```suggestion
- observability and traceability,
```

File: website/content/docs/intro/usecases/what-is-a-service-mesh.mdx
Comment (blake) [inline website/content/docs/intro/usecases/what-is-a-service-mesh.mdx:5]:
```suggestion
  Learn what a service mesh is, it's benefits, and how it works.
```

File: website/content/docs/intro/usecases/what-is-a-service-mesh.mdx
Comment (blake) [inline website/content/docs/intro/usecases/what-is-a-service-mesh.mdx:28]:
```suggestion
A common use case for leveraging a _service mesh_ is to achieve a [_zero trust_ model](/use-cases/zero-trust-networking).
```

File: website/content/docs/intro/usecases/what-is-a-service-mesh.mdx
Comment (blake) [inline website/content/docs/intro/usecases/what-is-a-service-mesh.mdx:53]:
```suggestion
API Gateways are frequently deployed alongside a load balancer to ensure traffic is directed to healthy and available instances of the service.
The mesh reduces the load balancer footprint as routing responsibilities are handled in a decentralized manner.
```

File: website/content/docs/intro/usecases/what-is-a-service-mesh.mdx
Comment (blake) [inline website/content/docs/intro/usecases/what-is-a-service-mesh.mdx:55]:
```suggestion
API Gateways can be used together with a _service mesh_ to bridge external networks (non-mesh) with a _service mesh_.
```

File: website/content/docs/intro/usecases/what-is-a-service-mesh.mdx
Comment (blake) [inline website/content/docs/intro/usecases/what-is-a-service-mesh.mdx:57]:
```suggestion
-> **Note**: API Gateways are frequently used to accept north-south based traffic. North-south traffic is networking traffic that either enters or exits a data center or a virtual private network (VPC).
```

File: website/content/docs/intro/usecases/what-is-a-service-mesh.mdx
Comment (blake) [inline website/content/docs/intro/usecases/what-is-a-service-mesh.mdx:59]:
```suggestion
A _service mesh_ is primarily used for handling east-west based traffic. East-west traffic traditionally remains inside a data center or a VPC.
A _service mesh_ can be connected to another _service mesh_ in another data center or VPC to form a federated mesh.
```

File: website/content/docs/intro/usecases/what-is-a-service-mesh.mdx
Comment (blake) [inline website/content/docs/intro/usecases/what-is-a-service-mesh.mdx:72]:
```suggestion
Implementing dynamic network behavior is critical as more and more applications are deployed across different cloud providers (multi-cloud) and private data centers.
```

**Review by blake — COMMENTED** [review]
I took a quick first pass at reviewing this document. I'll review it in more detail later on, and will let you know if I have additional suggestions.

File: website/content/docs/intro/usecases/what-is-a-service-mesh.mdx
Comment (karl-cardenas-coding) [reply website/content/docs/intro/usecases/what-is-a-service-mesh.mdx:15]: We normally use italic for words that standout, such as service names. After reviewing the style guide, we only need to use italic in the [first usage of the term](https://docs.google.com/document/d/1MRvGd6tS5JkIwl_GssbyExkMJqOXKeUE00kSEtFi8m8/edit#heading=h.lhe5v722498w).

**Comment by jon-kow** [comment]
Under this section 
How does a Service Mesh work?
A service mesh typically consist of a control plane, and a data plane. The control plane maintains a central registry that keeps track of all services and their respective IP addresses, this is called service discovery. 

Can we link service discovery to https://www.hashicorp.com/products/consul/service-discovery-and-health-checking

File: website/content/docs/intro/usecases/what-is-a-service-mesh.mdx
Comment (jkirschner-hashicorp) [inline website/content/docs/intro/usecases/what-is-a-service-mesh.mdx:97]: What is "multi-networking"? Feels like an undefined buzzword.

Comment (jkirschner-hashicorp) [reply website/content/docs/intro/usecases/what-is-a-service-mesh.mdx:97]: I feel like we're missing something about heterogenous environments in here (e.g., multi-cloud, hybrid-cloud ... whatever combo you have)

File: website/content/docs/intro/usecases/what-is-a-service-mesh.mdx
Comment (jkirschner-hashicorp) [inline website/content/docs/intro/usecases/what-is-a-service-mesh.mdx:99]: I don't know that it's obvious to the reader that Consul doesn't leave it as an exercise to the reader to figure out how to use a data plane with it. We integrate closely with a commonly-used, open source service mesh data plane: Envoy.

File: website/content/docs/intro/usecases/what-is-a-service-mesh.mdx
Comment (jkirschner-hashicorp) [inline website/content/docs/intro/usecases/what-is-a-service-mesh.mdx:101]: Should we include ECS/serverless and bare metal?

Comment (jkirschner-hashicorp) [reply website/content/docs/intro/usecases/what-is-a-service-mesh.mdx:101]: Also, thanks for including Nomad :)

**Review by jkirschner-hashicorp — COMMENTED** [review]
@karl-cardenas-coding : was able to get through most of a review. I made an attempt at simplifying a few sections - feel free to pick-and-choose what of those rewrites you want to apply!

**Comment by srikrishmurthy** [comment]
@karl-cardenas-coding @jon-kow -This is a nice article. the note that users can get started quickly with HCP is called out at the end.. I would suggest we call that out early in the doc. For context: I spoke with a few enterprises in which the architects/eng directors are exploring service mesh and one of them stood up a consul cluster. During our discussion he called out the effort required to bootstrap the cluster. I mentioned about HCP and he called out he was not aware that he could bootstrap in a much quicker/simpler way..

**Comment by jkirschner-hashicorp** [comment]
> I mentioned about HCP and he called out he was not aware that he could bootstrap in a much quicker/simpler way..

@srikrishmurthy: are we certain that it's obvious that one of the benefits of "HCP (Cloud) Consul" is that you can bootstrap in a much quicker/simpler way? Or do we need to make that value more explicit (in various places)?

File: website/content/docs/intro/usecases/what-is-a-service-mesh.mdx
Comment (jon-kow) [reply website/content/docs/intro/usecases/what-is-a-service-mesh.mdx:19]: Some of these it would make sense to link out to other pages, but at the same time we don't want to overwhelm a reader with links.

File: website/content/docs/intro/usecases/what-is-a-service-mesh.mdx
Comment (jon-kow) [reply website/content/docs/intro/usecases/what-is-a-service-mesh.mdx:12]: @jkirschner-hashicorp This would be another question or headline. It would generate more traffic from those searching for "microservices architecture patterns." If this isn't too much feel free to include this.

**Comment by karl-cardenas-coding** [comment]
> > I mentioned about HCP and he called out he was not aware that he could bootstrap in a much quicker/simpler way..
> 
> @srikrishmurthy: are we certain that it's obvious that one of the benefits of "HCP (Cloud) Consul" is that you can bootstrap in a much quicker/simpler way? Or do we need to make that value more explicit (in various places)?

If the intent is to educate the reader about service mesh. Then I feel like we should keep the outline as is, and focus on calling out HCP at the end. Otherwise, this might feel like a marketing article. I'll defer to @jon-kow  as he is the SEO expert 😄

**Comment by jon-kow** [comment]
@srikrishmurthy I agree with @karl-cardenas-coding. The point of this page is to educate users and not just sell them on our product. If we turn this into a marketing article it will likely hurt us for rankings since engagement would likely drop on the page.

File: website/content/docs/intro/usecases/what-is-a-service-mesh.mdx
Comment (karl-cardenas-coding) [reply website/content/docs/intro/usecases/what-is-a-service-mesh.mdx:6]: Exactly, yup :)

File: website/content/docs/intro/usecases/what-is-a-service-mesh.mdx
Comment (karl-cardenas-coding) [reply website/content/docs/intro/usecases/what-is-a-service-mesh.mdx:40]: @jon-kow I know you wanted a certain work count but are you comfortable with this suggestion? I like the suggestion a lot 🚀

File: website/content/docs/intro/usecases/what-is-a-service-mesh.mdx
Comment (karl-cardenas-coding) [reply website/content/docs/intro/usecases/what-is-a-service-mesh.mdx:60]: We can probably do without that statement, but I do want to keep the Note that explains north-south vs east-west.

File: website/content/docs/intro/usecases/what-is-a-service-mesh.mdx
Comment (karl-cardenas-coding) [reply website/content/docs/intro/usecases/what-is-a-service-mesh.mdx:11]: I like it!

File: website/content/docs/intro/usecases/what-is-a-service-mesh.mdx
Comment (karl-cardenas-coding) [reply website/content/docs/intro/usecases/what-is-a-service-mesh.mdx:101]: I don't feel comfortable stating serverless because that is not a first class citizen usecase. ECS, I view as a container orchestration platform.

File: website/content/docs/intro/usecases/what-is-a-service-mesh.mdx
Comment (karl-cardenas-coding) [reply website/content/docs/intro/usecases/what-is-a-service-mesh.mdx:62]: I've answered this question more times than I can count. What helps people understand API Gateways and how Consul is different is the east-west/north-south explanation.  For that reason, I believe we have to keep that information here.

File: website/content/docs/intro/usecases/what-is-a-service-mesh.mdx
Comment (karl-cardenas-coding) [reply website/content/docs/intro/usecases/what-is-a-service-mesh.mdx:91]: @jon-kow thoughts on this? Grammatically, it is a correct suggestion. I'm just not sure from an SEO perspective.

File: website/content/docs/intro/usecases/what-is-a-service-mesh.mdx
Comment (trujillo-adam) [inline website/content/docs/intro/usecases/what-is-a-service-mesh.mdx:60]:
```suggestion
-> **API gateways and traffic direction**: API gateways are often used to accept north-south traffic. North-south traffic is networking traffic that either enters or exits a data center or a virtual private network (VPC).
A service mesh is primarily used for handling east-west traffic. East-west traffic traditionally remains inside a data center or a VPC.
A service mesh can be connected to another service mesh in another data center or VPC to form a federated mesh.
```
Trying to make a more meaningful label for the callout. I also removed "based" because in order to properly punctuate, you would need to write "north-south-based",  which is kind of weird.

File: website/content/docs/intro/usecases/what-is-a-service-mesh.mdx
Comment (trujillo-adam) [reply website/content/docs/intro/usecases/what-is-a-service-mesh.mdx:91]: I believe that the proper grammar would be to use lowercase for "platform". Hyphenating words technically makes them one word (except for when used for phrasal adjectives), so only the first letter should be capped.

**Review by trujillo-adam — APPROVED** [review]
Few comments, questions, suggestions. Nothing critical - nice work!

File: website/content/docs/intro/usecases/what-is-a-service-mesh.mdx
Comment (karl-cardenas-coding) [reply website/content/docs/intro/usecases/what-is-a-service-mesh.mdx:50]: Yup, managing services gets a bit sticky as that is what an orchestrator does.

File: website/content/docs/intro/usecases/what-is-a-service-mesh.mdx
Comment (karl-cardenas-coding) [reply website/content/docs/intro/usecases/what-is-a-service-mesh.mdx:30]: Good point. I added the following text. 

In traditional security strategies, protection is primarily focused at the perimeter of a network. 
In cloud environments, the surface area for network access is much wider than the traditional on-premises networks. 
In addition, traditional security practices overlook the fact that many bad actors can originate from within the network walls. 
A zero trust model addresses these concerns while allowing organizations to scale as needed.

File: website/content/docs/intro/usecases/what-is-a-service-mesh.mdx
Comment (karl-cardenas-coding) [reply website/content/docs/intro/usecases/what-is-a-service-mesh.mdx:87]: What do you have in mind here @jkirschner-hashicorp? Because I thought I addressed it below.

File: website/content/docs/intro/usecases/what-is-a-service-mesh.mdx
Comment (karl-cardenas-coding) [reply website/content/docs/intro/usecases/what-is-a-service-mesh.mdx:98]: Adding hybrid cloud, good call.

File: website/content/docs/intro/usecases/what-is-a-service-mesh.mdx
Comment (karl-cardenas-coding) [reply website/content/docs/intro/usecases/what-is-a-service-mesh.mdx:102]: @jkirschner-hashicorp  the term "multi-networking" is something Neena has been wanting us to use. I would check with her, but you're right. I don't think there is a clear definition of the term. 

I'm adding multi-cloud, hybrid-cloud

File: website/content/docs/intro/usecases/what-is-a-service-mesh.mdx
Comment (karl-cardenas-coding) [reply website/content/docs/intro/usecases/what-is-a-service-mesh.mdx:104]: @jkirschner-hashicorp Good call Jared. How does this sound. I added it after the last sentence.

The data plane is supported by Consul through its first class support of [Envoy](https://www.envoyproxy.io/) as a proxy.

**Review by jon-kow — APPROVED** [review]

**Comment by hc-github-team-consul-core** [comment]
:cherries: If backport labels were added before merging, cherry-picking will start automatically.

To retroactively trigger a backport after merging, add backport labels and re-run https://circleci.com/gh/hashicorp/consul/576014.

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit b09272f361abe54d0e4fda4e48f6b08b2d7fa31b onto `stable-website` succeeded!

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit b09272f361abe54d0e4fda4e48f6b08b2d7fa31b onto `release/1.11.x` succeeded!

**Pull request merged**
